### PR TITLE
refactor: replace old icon with new

### LINKED ISSUE #1470: Replace webmanifest logo's with new logo
**Is your feature request related to a problem? Please describe.**
It's very nice to use a consistent logo. We have a new one with two Z's in it and a more generally accepted color scheme.

**Describe the solution you'd like**
Replace the current icons in webmanifest with new logo files. There's tools to automatically generate a whole set.

**Describe alternatives you've considered**
Keep it the way it is. Just a pity... we have a betyer logo - why not use it?

**Additional context**
Logo files on request.

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
resolves #1470

### CODE REVIEW

**Comment by vinaybadgujar102** [comment]
Hey @tijlleenders, please review and suggest changes to make

File: vite.config.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default defineConfig({
         name: "ZinZen: Deliver purpose",
         icons: [
           {
-            src: "pwa-192x192.png",
+            src: "pwa-192x192-new.png",
```
Comment (tijlleenders) [inline vite.config.ts:40]: @vinaybadgujar102 Why rename the src if we're not using the old file anymore?

File: public/mstile-150x150.png
Comment (tijlleenders) [inline public/mstile-150x150.png:1]: @vinaybadgujar102 Shouldn't this have dimensions 150x150 as stated in the filename?

File: public/pwa-512x512.png
Comment (tijlleenders) [inline public/pwa-512x512.png:1]: @vinaybadgujar102 Why is the new file not filling the 512x512 area?

**Review by tijlleenders — CHANGES_REQUESTED** [review]
It all looks good except some questions about sizes.
I don't know if that's related to the generator/program you used or if you did things manually.

https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs seems like a pretty recently updated exhaustive list of (fav)icons... but if you just regenerate the ones mentioned in review we can merge this.

File: public/pwa-512x512.png
Comment (vinaybadgujar102) [reply public/pwa-512x512.png:1]: I replaced it with the file which fills the full area, but as you can see the logo is not fully visible. The borders are cropped in icon tile. Same was happening with our previous icon.
WhatsApp Image 2023-09-20 at 10 55 42 AM
I have pushed new commit with new files, you can check it out.

File: public/mstile-150x150.png
Comment (vinaybadgujar102) [reply public/mstile-150x150.png:1]: this file was auto-generated using pwa-generator. The previous icon also had same dimensions.

File: vite.config.ts (hunk 1 above)
Comment (vinaybadgujar102) [reply vite.config.ts]: resolved

File: public/mstile-150x150.png
Comment (vinaybadgujar102) [reply public/mstile-150x150.png:1]: According to the article you shared, now there is no need to create separate files for Microsoft tile support.

> Microsoft Edge used to [support](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/hh772707(v=vs.85)) a special icon format to pin websites to the start menu. For recent versions of Windows, this is no longer required.

Same applies for safari-pinned-icon.

File: public/pwa-512x512.png
Comment (tijlleenders) [reply public/pwa-512x512.png:1]: Well if this is the effect in practice then you were right to make it smaller!

**Review by tijlleenders — APPROVED** [review]

**Pull request merged**